### PR TITLE
[container][m]: merged DataPackageView and TabularResourceView containers - refs #67

### PR DESCRIPTION
**Pull request by anuveyatsu**

### CODE REVIEW

File: __tests__/src/containers/DataPackageViewContainer.test.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
 import "babel-polyfill"
+//libs:
 import React from "react"
 import { shallow, mount } from 'enzyme' //for testing with shallow/mount wrapper
 import sinon from 'sinon' //for spy
+//components:
 import { DataPackageViewContainer } from "../../../src/containers/DataPackageViewContainer"
 import ContainerWithRedux from "../../../src/containers/DataPackageViewContainer"
+import { TabularResourceViewContainer } from "../../../src/containers/TabularResourceViewContainer"
```
Comment (subhankarb) [inline __tests__/src/containers/DataPackageViewContainer.test.js]: What is this for. I think we are not using it

File: __tests__/src/containers/DataPackageViewContainer.test.js [hunk 2]
```diff
@@ -73,19 +77,15 @@ describe("Datapackage View Container", () => {
   const wrapper = shallow(<DataPackageViewContainer idx={0} />)
 
   it("should render PlotlyChart component and pass index", () => {
-    expect(wrapper.props().idx).toEqual(0)
-    expect(wrapper.text()).toEqual('<PlotlyChart />')
+    expect(wrapper.childAt(0).props().idx).toEqual(0)
+    expect(wrapper.text()).toContain('<PlotlyChart />')
```
Comment (subhankarb) [inline __tests__/src/containers/DataPackageViewContainer.test.js:80]: I think when we are using react we can use ```(wrapper.find(<PlotlyChart/>).length).tograterThan(0)```

Comment (subhankarb) [inline __tests__/src/containers/DataPackageViewContainer.test.js:80]: I think when we are using react we can use ```(wrapper.find(<PlotlyChart/>).length).tograterThan(0)```

**Review by subhankarb — APPROVED** [review]

File: __tests__/src/containers/DataPackageViewContainer.test.js (hunk 1 above)
Comment (anuveyatsu) [reply __tests__/src/containers/DataPackageViewContainer.test.js]: Will delete this line

**Pull request merged**